.scalafmt.conf

```diff
@@ -0,0 +1,60 @@
+version = 3.7.2
+runner.dialect = scala213
+maxColumn = 150
+lineEndings = unix
+importSelectors = singleLine
+rewriteTokens = {
+  "⇒": "=>"
+  "→": "->"
+  "←": "<-"
+}
+
+project {
+  git = true
+}
+
+align {
+  preset = more
+  tokens = [
+    "extends",
```

Comment:
Add `":"` to these tokens to maintain alignment of type annotations in parameter lists.